docs/README.md

```diff
@@ -1,19 +1,21 @@
-# webots-doc
+# Webots Documentation
 
-This repository contains the documentation of the Webots software.
+This repository contains the documentation for the Webots open source simulation software.
 
-The master branch of this repository is in production on the Cyberbotics website:
+The master branch of this repository is in production on the Cyberbotics website, which you can access by visiting the following links:
 
 - https://www.cyberbotics.com/doc/guide/index
 - https://www.cyberbotics.com/doc/reference/index
 - https://www.cyberbotics.com/doc/blog/index
 - https://www.cyberbotics.com/doc/automobile/index
 
-You are very welcome to contribute to make this documentation better!
-In order to proceed, simply fork this repository, make your modifications and
-open a pull request that we will review and merge.
+You are very welcome to contribute to make this documentation better.
+In order to proceed, follow these steps:
+1. Simply fork this repository 
```

Comment:
```suggestion
1. Fork this repository 
```

Reply:
Thanks for pointing that out . I have tried to correct this as per the requirement.